contrib/sandbox/packages/st2-mistral/actions/transformer.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+import six
+import yaml
+
+from st2actions.runners import pythonrunner
+from st2actions.runners.mistral import utils
+from st2client import client
+
+
+class MistralDSLTransformer(pythonrunner.Action):
```

Comment:
Why does the impl here and in  st2actions/st2actions/runners/mistral/utils.py look so similar? Perhaps you can call into the utils from here?

Also, what is the purpose of this action? Could the actions have been a separate PR or at least a separate commit? 

Btw, we typically put sandbox action in st2incubator now.